.github/workflows/ci.yml

```diff
@@ -14,31 +14,31 @@ permissions:
 
 jobs:
   docs:
-    uses: rspec/rspec/.github/workflows/documentation_coverage.yml@main
+    uses: benoittgt/rspec-1/.github/workflows/documentation_coverage.yml@ruby-3-4
```

Comment:
To be reverted. :) 

I had no push right on rspec/rspec so I used a fork. :)

Reply:
I will look into that.